database/src/main/resources/db/migration/V6__create_food_schema.sql

```diff
@@ -31,22 +31,19 @@ CREATE TABLE IF NOT EXISTS restaurant_cuisine(
     PRIMARY KEY (restaurant_id, cuisine)
```

Comment:
1.Nowa migracyja musi być, to nam nie zadziała, bo już schema istnieje.
2.Nazwijmy ją więc V8__food_schema_remove_dish_add_mark.sql
3.Tam usuwamy tabelę dish i dodajemy mark_restaurant.